### DISCUSSION TITLE
How to share the playground screen?

### DISCUSSION THREAD

**Discussion by SionicAI-Engineering** [langflow-ai/langflow#2603] [Q&A]
I want to share the Playground screen like a chat widget and would like to know if this is supported.
If not, could you add it?

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
